fileio/private/read_nex5.m

```diff
@@ -0,0 +1,167 @@
+function [varargout] = read_nex5(filename, varargin)
+
+% READ_NEX5 reads header or data from a Nex Technologies *.nex5 file, 
+% which is a file containing action-potential (spike) timestamps and waveforms
+% (spike channels), event timestamps (event channels), and continuous
+% variable data (continuous A/D channels).
+%
+% LFP and spike waveform data that is returned by this function is 
+% expressed in microVolt.
+%
+% Use as
+%   [hdr] = read_nex5(filename)
+%   [dat] = read_nex5(filename, ...)
+%   [dat1, dat2, dat3, hdr] = read_nex5(filename, ...)
+%
+% Optional arguments should be specified in key-value pairs and can be
+%   header      structure with header information
+%   feedback    0 or 1
+%   tsonly      0 or 1, read only the timestamps and not the waveforms
+%   channel     number, or list of numbers (that will result in multiple outputs)
+%   begsample   number (for continuous only)
+%   endsample   number (for continuous only)
+%
+% See also READ_NEX5_HEADER
+%
+% Copyright (C) 2020 Robert Oostenveld
```

Comment:
Did Robert write this? If not, feel free to add your own name here. Likewise for the other newly contributed code. Or, if the new code is based on some older code that Robert wrote, you can add both your names

Reply:
I wrote the code myself but is is based on Robert's read_nex_* code. I will use both names.

Reply:
Top! Nice to meet you online, Alex! Good to know who's the named person behind NeuroExplorer